app/models/course.rb

```diff
@@ -48,7 +48,10 @@ class Course < ApplicationRecord
     not_required: 9,
   }
 
-  enum :campaign_name, COURSE_CAMPAIGNS
+  enum :campaign_name, {
+    no_campaign: "no_campaign",
```

Comment:
Do we need this? The presence of `nil` on this field should indicate it isn't a part of a campaign?

Reply:
That is a bit problematic with the radio buttons and validations, we get into nil/false issues etc